src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -48,10 +48,11 @@ public static bool IsExpressionOrOntogenyFactor(this IParameter parameter)
 
       public static bool IsIndividualMolecule(this IParameter parameter)
```

Comment:
Should be renamed to ```IsIndividualMoleculeParameter```

Reply:
Disagree. Those method are extension method and read as 

parameter.IsIndividualMolecule()


Adding parameter at the end is 1/ redundant 2/ not in accordance with other methods in the same file such as IsExpression etc..


I would simply leave it as is

Reply:
+1 :)